src/ai/backend/manager/models/group.py

```diff
@@ -220,35 +229,26 @@ class Arguments:
         lambda name, props, **kwargs: (props.domain_name, None)
     )
     async def mutate(cls, root, info, name, props):
-        async with info.context['dbpool'].acquire() as conn, conn.begin():
-            assert _rx_slug.search(name) is not None, 'invalid name format. slug format required.'
-            data = {
-                'name': name,
-                'description': props.description,
-                'is_active': props.is_active,
-                'domain_name': props.domain_name,
-                'total_resource_slots': ResourceSlot.from_user_input(
-                    props.total_resource_slots, None),
-                'allowed_vfolder_hosts': props.allowed_vfolder_hosts,
-                'integration_id': props.integration_id,
-            }
-            query = (groups.insert().values(data))
-            try:
-                result = await conn.execute(query)
-                if result.rowcount > 0:
-                    checkq = groups.select().where((groups.c.name == name) &
-                                                   (groups.c.domain_name == props.domain_name))
-                    result = await conn.execute(checkq)
-                    o = Group.from_row(info.context, await result.first())
-                    return cls(ok=True, msg='success', group=o)
-                else:
-                    return cls(ok=False, msg='failed to create group', group=None)
-            except (pg.IntegrityError, sa.exc.IntegrityError) as e:
-                return cls(ok=False, msg=f'integrity error: {e}', group=None)
-            except (asyncio.CancelledError, asyncio.TimeoutError):
-                raise
-            except Exception as e:
-                return cls(ok=False, msg=f'unexpected error: {e}', group=None)
+        assert _rx_slug.search(name) is not None, 'invalid name format. slug format required.'
```

Comment:
Let's change assert into an if statement with explicit raising of exceptions.
(If Python is built with optimization flags, assert statements may be ignored)

Reply:
Replaced assert into explicit exceptions.